app/client/components/user/EditUserForm.tsx

```diff
@@ -193,11 +197,22 @@ const EditUserForm = (props: Props) => {
     <div style={{ display: "flex", flexDirection: "column" }}>
       <Form onFinish={onFinish} layout="vertical" form={form}>
         <br />
-
         <Card
           title={"User Details"}
           extra={
             <Space>
+              <Button
+                type="primary"
+                onClick={props.onViewHistory}
+                hidden={
+                  !(
+                    props.adminTier === "superadmin" ||
+                    props.adminTier === "sempoadmin"
+                  )
+                }
+              >
+                View History
+              </Button>
```

Comment:
I don't think the history button should be "primary" color
I think the button order should be:
  │           delete           │     │         history            │   │             save                │